manticore/platforms/linux.py

```diff
@@ -1292,12 +1292,12 @@ def sys_write(self, fd, buf, count):
             data = self._transform_write_data(data)
             write_fd.write(data)
 
-            for line in ''.join([str(x) for x in data]).split('\n'):
+            for line in b''.join([x for x in data]).split(b'\n'):
                 logger.debug("WRITE(%d, 0x%08x, %d) -> <%.48r>",
                              fd,
                              buf,
                              count,
-                             line)
+                             line.decode('latin-1'))
```

Comment:
what's the reason for latin-1?

Reply:
latin-1 will decode 0x00-0xff

Reply:
regular `line.decode()` seems to work fine for me. tbh i don't know a whole lot about encodings, is there any reason we shouldn't just use utf-8?

Reply:
utf-8 it would raise with `UnicodeDecodeError` on any byte > 0x7f and in this case what is being written is just binary data. would be nice to replace this at a later date with something similar of `xxd` output. but that might be too verbose

Reply:
```
>>> b'sdkjfhk\x20\x8f'.decode()
Traceback (most recent call last):
  File "<input>", line 1, in <module>
    b'sdkjfhk\x20\x8f'.decode()
UnicodeDecodeError: 'utf-8' codec can't decode byte 0x8f in position 8: invalid start byte
>>> b'sdkjfhk\x20\x8f'.decode('latin-1')
'sdkjfhk \x8f'
>>>
```